build.gradle

```diff
@@ -208,7 +208,7 @@ ext.libraries = [
 ext["rest-assured.version"] = '4.1.2'
 
 dependencies {
-  implementation group: 'com.azure', name: 'azure-storage-blob', version: '12.1.0'
+  implementation group: 'com.azure', name: 'azure-storage-blob', version: '12.0.1'
```

Comment:
why the downgrade?

Reply:
the azure-core-test library is not available for `12.1.0` version